### PR TITLE
first version of data load

### LINKED ISSUE #610: First commit test

### PR DESCRIPTION
**Pull request by zhxfl**
Resolve #610

### CODE REVIEW

File: fluid/DeepASR/data_utils/load_data.py [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+"""
+
+import random
+import Queue
+import numpy
+import struct
+import data_utils.trans_mean_variance_norm as trans_mean_variance_norm
+import data_utils.trans_add_delta as trans_add_delta
+
+g_lblock = []
```
Comment (kuke) [inline fluid/DeepASR/data_utils/load_data.py]: Please avoid using the global variables

File: fluid/DeepASR/data_utils/load_data.py [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+
+class OneBlock(object):
+    """ Documentation for a class.
+        struct for one block :
+        contain label, label desc, feature, feature_desc
+    """
+
+    def __init__(self):
+        """The constructor."""
+        self.label = ""
```
Comment (kuke) [inline fluid/DeepASR/data_utils/load_data.py]: For different types of variables, please use different types of initial value.

File: fluid/DeepASR/data_utils/load_data.py [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+def set_trans(ltrans):
+    global g_ltrans
+    g_ltrans = ltrans
+
+
+def load_list(sfeature_lst, slabel_lst):
+    """ load list """
+    global g_lblock
+
+    lFeature = open(sfeature_lst).readlines()
```
Comment (kuke) [inline fluid/DeepASR/data_utils/load_data.py]: Please unify the format to name variables

File: fluid/DeepASR/stacked_dynamic_lstm.py [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+    exe = fluid.Executor(place)
+    exe.run(fluid.default_startup_program())
+
+    ltrans = [
+        trans_add_delta.TransAddDelta(2, 2),
+        trans_mean_variance_norm.TransMeanVarianceNorm(
+            "data/global_mean_var_search26kHr"), trans_splice.TransSplice()
+    ]
+    load_data.set_trans(ltrans)
+    load_data.load_list("/home/disk2/mini_speech_fbank_40/data/feature.lst",
```
Comment (kuke) [inline fluid/DeepASR/stacked_dynamic_lstm.py]: Use arguments instead of hard-coded strings. The same below.

**Review by kuke — COMMENTED** [review]
Please pay more attention to the coding style. Thanks

File: fluid/DeepASR/data_utils/load_data.py [hunk 5]
```diff
@@ -0,0 +1,195 @@
+#by zhxfl 2018.01.24
```
Comment (pkuyym) [inline fluid/DeepASR/data_utils/load_data.py]: Please remove this comment.

File: fluid/DeepASR/data_utils/trans_add_delta.py [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+
+class TransAddDelta(object):
+    """ add delta of feature data 
+        trans feature for shape(a, b) to shape(a, b * 3)
+    """
+
+    def __init__(self, norder=2, nwindow=2):
+        """ init construction
+            Args:
+                norder: default 2
```
Comment (pkuyym) [inline fluid/DeepASR/data_utils/trans_add_delta.py:18]: Please explain the meaning of arguments.

File: fluid/DeepASR/data_utils/load_data.py (hunk 1 above)
Comment (zhxfl) [reply fluid/DeepASR/data_utils/load_data.py]: done

File: fluid/DeepASR/data_utils/load_data.py (hunk 2 above)
Comment (zhxfl) [reply fluid/DeepASR/data_utils/load_data.py]: done

File: fluid/DeepASR/stacked_dynamic_lstm.py (hunk 4 above)
Comment (zhxfl) [reply fluid/DeepASR/stacked_dynamic_lstm.py]: done

File: fluid/DeepASR/data_utils/load_data.py (hunk 5 above)
Comment (zhxfl) [reply fluid/DeepASR/data_utils/load_data.py]: done

**Review by kuke — APPROVED** [review]
LGTM

File: fluid/DeepASR/data_utils/trans_add_delta.py (hunk 6 above)
Comment (zhxfl) [reply fluid/DeepASR/data_utils/trans_add_delta.py:18]: done

File: fluid/DeepASR/data_utils/load_data.py (hunk 3 above)
Comment (zhxfl) [reply fluid/DeepASR/data_utils/load_data.py]: done

**Pull request merged**